scripts/ci/requirements.txt

```diff
@@ -14,7 +14,7 @@ flake8
 flask
 flask-admin
 flask-cache
-flask-login
+flask-login==0.2.11
```

Comment:
Can you rebase off of master to make this change go away (since IIRC it was in your other PR)?